ROADMAP.md

```diff
@@ -0,0 +1,55 @@
+
+# Binaryen Roadmap
+
+## Planned Work
+
+* Code annotations (inlining hints, etc.)
+
+## Ongoing Work
+
+* [Improve compile times]
+* New interpreter (avoid exceptions, enable stack switching)
+
+## Completed Work
+
+* Wasm features support:
+  * Atomics
+  * MutableGlobals
+  * TruncSat
+  * SIMD
+  * BulkMemory
+  * SignExt
+  * ExceptionHandling
+  * TailCall
+  * ReferenceTypes
+  * Multivalue
+  * GC
+  * Memory64
+  * RelaxedSIMD
+  * ExtendedConst
+  * Strings
+  * MultiMemory
+  * StackSwitching
```

Comment:
The stack switching support is not quite complete. In particular, the validation is not implemented.

Reply:
Thanks, good point.

Also SharedEverything is still under development at the spec level, so that should be under Ongoing Work I suppose.

If we decide to land this, I'll update those parts first (for now the larger question for me is if this document seems useful at all).